include/rocksdb/wide_columns.h

```diff
@@ -218,4 +220,44 @@ inline bool operator!=(const PinnableWideColumns& lhs,
   return !(lhs == rhs);
 }
 
+// PinnableWideColumns bundled with ColumnFamilyHandle and Status for the query.
+// Used for the results of wide-column queries grouped by Column Families
+class PinnableWideColumnsBundle {
+ public:
+  ColumnFamilyHandle* column_family() const { return column_family_; }
+  const Status& status() const { return status_; }
+  const PinnableWideColumns& pinnable_wide_columns() const {
```

Comment:
How about `result()` or `columns()`? (`result()` might be a bit better because `PinnableWideColumns` itself has a method called `columns()`.) We might also want to name the member variable accordingly.

Reply:
Yeah, I wanted to make it distinguishable with `PinnableWideColumns`'s `columns()`. I will change it `result()`

Reply:
Or `attributes()`? if we are going to name the class `AttributeGroup`

Reply:
`attributes()` sounds great to me!

Reply:
One more idea here, LMK what you think: we could call the method `columns()` and make it relay the result of the PWC's `columns()`.